diffpy/snmf/stretchednmfapp.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import argparse
+from pathlib import Path
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(
+        prog="stretched_nmf",
+        description="Stretched Nonnegative Matrix Factorization"
+    )
+    parser.add_argument('-i', '--input-directory', type=str, default=None,
+                        help="Directory containing experimental data. Defaults to current working directory.")
+    parser.add_argument('-o', '--output-directory', type=str,
+                        help="The directory where the results will be written. Defaults to '<input_directory/snmf_results>'.")
```

Comment:
this should be `<input_directory>/snmf_results` i.e., replace `<input_directory>`  with the actual input directory, but it will have a folder hanging off it which is actually called "snmf_results" by default

Reply:
I am a little confused. How should I replace <input_directory> if it's not known. Do you mean that the` output_directory` will always be `<input_directory>/snmf_results`?

Reply:
I just want you to replace `<input_directory/snmf_results>`, which is exactly what you wrote, with exactly `<input_directory>/snmf_results`.

Reply:
I understand. I will do that